### DISCUSSION TITLE
Параметры методов MVC контроллера

### DISCUSSION THREAD

**Discussion by golosovk** [ZeLebo/Timetable#62] [General]
https://github.com/ZeLebo/Timetable/blob/01153ef1a895f5c73b2f97d99bec9ec99d4c96c2/src/main/kotlin/nsu/controllers/timetable/TimetableController.kt#L52
Параметр метода может быть и не `String`. Если предполагается что параметр будет целым числом, то и объявляйте его типом `Integer`. Не придется в теле метода конвертировать.

**Comment by ZeLebo** [comment]
Так всэ проще, это метод не для этого)))

Короче, тут будет другой контроллер
